packages/web-components/src/menu/menu.styles.ts

```diff
@@ -0,0 +1,19 @@
+import { css } from '@microsoft/fast-element';
+import { display } from '@microsoft/fast-foundation';
+import {} from '../theme/design-tokens.js';
+
+/** Menu styles
+ * @public
+ */
+export const styles = css`
+  ${display('block')}
```

Comment:
I'm not sure we want the component to default to block layout - @miroslavstastny, thoughts here?